src/main/resources/application.properties

```diff
@@ -44,13 +43,11 @@ quarkus.cache.caffeine.default.maximum-size=1000
 quarkus.cache.caffeine.default.expire-after-write=30M
 
 sources.beacon=true
-
 datasets.filters=access_rights,theme,tags,organization,res_format
-
 %dev.quarkus.oidc.auth-server-url=https://id.portal.dev.gdi.lu/realms/gdi
 %dev.quarkus.oidc.client-id=gdi
 %dev.quarkus.oidc.credentials.secret=dummy-secret
-%dev.quarkus.rest-client.ckan_yaml.url=https://catalogue.portal.dev.gdi.lu/
+%dev.quarkus.rest-client.ckan_yaml.url=http://localhost:5500/
```

Comment:
important: please revert this change.

It's simplifies local development, as can simply start and play with coding.

Reply:
Sorry this one, reverted it. I think I staged it by accident 🙈